README.md

```diff
@@ -101,35 +106,9 @@ Also, on Windows, `webview.dll` and `WebView2Loader.dll` must be placed into the
 
 To cross-compile a webview app - use [xgo](https://github.com/karalabe/xgo).
 
-### Known issues
-
-#### Accessing localhost on Windows
-
-If Edge (Chromium) isn't installed on the target machine, webview will use a UWP application context which disallows loopback by default. To enable it you need to run the following command from a command prompt with admin priviledges:
-
-```sh
-CheckNetIsolation.exe LoopbackExempt -a -n="Microsoft.Win32WebViewHost_cw5n1h2txyewy"
-```
-
-For app distribution we recommend automating this in your installer.
-
-### Migrating from v0.1.1 to v0.10.0
-
-1. `webview.Open()` has been removed. Use other webview APIs to create a window, open a link and run main UI loop.
-2. `webview.Debug()` and `webview.Debugf()` have been removed. Use your favorite logging library to debug webview apps.
-3. `webview.Settings` struct has been removed. Title, URL and size are controlled via other API setters and can be updated at any time, not only when webview is created.
-4. `Webview.Loop()` has been removed. Use `Run()` instead.
-5. `WebView.Run()`, `WebView.Terminate()`, `WebView.SetTitle()`, `WebView.Dispatch()` stayed the same.
-6. `WebView.Exit()` has been renamed to `WebView.Destroy()`
-7. `WebView.SetColor()` and `WebView.SetFullScreen()` have been removed. Use `Window()` to get native window handle and probably write some Cgo code to adjust native window to your taste.
-8. `webview.Dialog` has been removed. But it is likely to be brought back as a standalone module.
-9. `WebView.Eval()` remained the same.
-10. `WebView.InjectCSS()` has been removed. Use eval to inject style tag with CSS inside.
-11. `WebView.Bind()` kept the name, but changed the semantics. Only functions can be bound. Not the structs, like in Lorca.
-
```

Comment:
Do you think migration guides should be kept somewhere? Maybe we could use a `CHANGELOG.md` file generally, and include this information at first.